src/main/java/org/wso2/carbon/esb/connector/SendSMS.java

```diff
@@ -152,36 +147,33 @@ public void connect(MessageContext messageContext) throws ConnectException {
             handleException("IO error occur" + e.getMessage(), e, messageContext);
         }
     }
+
     /**
-     * Prepare pay load
+     * Prepare payload is used to delete the element in existing body and add the new element.
      *
-     * @param messageContext The message context that is processed by a handler in the handle method
-     * @param element        OMElement
+     * @param messageContext The message context that is used to prepare payload message flow.
+     * @param element        The OMElement that needs to be added in the body.
      */
-    private void preparePayload(MessageContext messageContext, OMElement element) {
+    public static void preparePayload(MessageContext messageContext, OMElement element) {
```

Comment:
It's wrong to make a instance method public static. unless there it's a util

Reply:
Done